src/Compiler/NodeVisitor/ChangeFiltersNodeVisitor.php

```diff
@@ -31,9 +30,12 @@ final class ChangeFiltersNodeVisitor extends NodeVisitorAbstract implements Actu
     /** @var array<string, string|array{string, string}|array{object, string}|callable> */
     private array $filters;
 
-    public function __construct(CompilerInterface $compiler)
+    /**
+     * @param array<string, string|array{string, string}|array{object, string}|callable> $filters
+     */
+    public function __construct(array $filters)
     {
-        $this->filters = $compiler->getFilters();
+        $this->filters = LatteVersion::isLatte2() ? array_change_key_case($filters) : $filters;
```

Comment:
Isn't this already covered on caller level?

Reply:
Unfortunatelly it isn't. I can’t find one place where to add it

Reply:
It should be in getFilters of LatteCompiler I think. From there all other services shuld get it